srcpkgs/pacman/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'pacman'
+pkgname=pacman
+version=5.2.1
+revision=1
+wrksrc="pacman-${version}"
+make_dirs="/var/lib/pacman 0755 root root
+ /usr/var/cache/pacman/pkg 0755 root root
+ /usr/share/libalpm/hooks 0755 root root
+ /usr/share/pacman/keyrings 0755 root root"
```

Comment:
Do we really want this? users shouldn't use pacman on a void linux root right? Are those directories required for chroot installations using pacman?

Reply:
Err, my bad, that last line isn't needed.

Reply:
You might want the keyrings directory in order to package the archlinux-keyring. Then `pacman-key --populate $keyring_name` will merge the keyring into /etc/pacman.d/gnupg and lsign the master keys.

If you're going to bootstrap archlinux from void, then pacman --root /path/to/chroot -Sy base_sytsem_packages is going to check those keys...

Reply:
@eli-schwartz I guess my acceptance for this PR was wrong?
I'm pretty much prefer to have you as maintainer for `pacman`.
Since you're obviously known pacman better than other people.

Reply:
To be honest, the keyrings/ directory isn't a direct requirement of pacman-key and it can be created just fine by e.g. an `archlinux-keyring` package or anything else which packages actual data there.